src/code/fv-components/fv-gene-label.js

```diff
@@ -0,0 +1,52 @@
+import React, {PropTypes} from 'react';
+
+const FVGeneLabelView = ({allele, chromosomeDescriptor, chromosomeHeight=122, stripe}) => {
+    var chromStart;
+    Object.keys(BioLogica.Species.Drake.geneList).forEach((geneName) => {
+      const gene = BioLogica.Species.Drake.geneList[geneName];
+      if (gene.alleles.indexOf(allele) > -1) {
+        chromStart = gene.start;
+      }
+    });
+    let bioChromHeight = BioLogica.Species.Drake.chromosomesLength[chromosomeDescriptor.name],
+        stripeHeight = chromosomeHeight * .03,
+        percentHeight = chromStart/bioChromHeight;
+
+    // Manually adjust certain labels and stripes up for the time being
+    let normalizedAllele = allele.toLowerCase();
+    if (normalizedAllele === "fl" || normalizedAllele === "hl" || normalizedAllele.indexOf('a') === 0) {
+      percentHeight -= .1;
+    }
+
+    const style = {marginTop: percentHeight * chromosomeHeight - stripeHeight * 3 + "px"},
+          stripeStyle = {height: stripeHeight + "px"};
+
+    let labelStyle = {marginTop: -stripeHeight * .8},
+        lineStyle = Object.assign({}, labelStyle);
+    
+    if (normalizedAllele === "fl") {
+      labelStyle.marginTop -= stripeHeight * 5;
+      lineStyle.marginTop -= stripeHeight;
+    }
```

Comment:
We talked this afternoon about the interplay between JS and CSS where line/label-drawing is concerned.  I'm concerned that this will lead to brittle code with higher-than-optimal coupling between the JS/CSS. Unfortunately, I don't have a clear idea of what a better solution would look like so I'm inclined to let this go for now with the proviso that as you work with the code moving forward and as you add support for other alleles that require special-casing, that you consider whether there's a way to consolidate more (all?) of the responsibility on one side of the fence or the other.